Ui/Component/Listing/Columns/CrossBorderClassActions.php

```diff
@@ -57,6 +67,20 @@ public function prepareDataSource(array $dataSource)
     {
         if (isset($dataSource['data']['items'])) {
             foreach ($dataSource['data']['items'] as &$item) {
+
+                /**
+                 * overriding cross border type id int with the label
+                 */
+                if(isset($item[CrossBorderClassInterface::CROSS_BORDER_TYPE])) {
+                    try {
+                        $item[CrossBorderClassInterface::CROSS_BORDER_TYPE] = $this->fetchCrossBorderTypeValue(
+                            $item[CrossBorderClassInterface::CROSS_BORDER_TYPE]
+                        );
+                    } catch (LocalizedException $e) {
+                        $item[CrossBorderClassInterface::CROSS_BORDER_TYPE] = CrossBorderClassInterface::NO_CROSS_BORDER_TYPE_TEXT;
```

Comment:
Limit line length to 120 characters

Reply:
Although this does exceed the line length limit, it is properly formatted. If this is an issue, we should discuss changes to the IDE formatter.